client/src/components/HistoryViewer/HistoryViewer.js

```diff
@@ -257,48 +209,45 @@ class HistoryViewer extends Component {
       schemaUrl: schemaUrl.replace(schemaSearch, (match) => schemaReplacements[match]),
       version,
       compare,
-      compareModeAvailable: this.compareModeAvailable(),
+      compareModeAvailable: compareModeAvailable(),
       previewState,
-      recordClass: this.props.recordClass,
-      onAfterRevert: this.handleAfterRevert
+      recordClass,
+      onAfterRevert: handleAfterRevert
     };
-
     return (
       <ResizeAware
-        className={this.getContainerClasses()}
-        onResize={({ width }) => this.props.onResize(width)}
+        className={getContainerClasses()}
+        onResize={({ width }) => onResize(width)}
       >
         <VersionDetailComponent {...props} />
       </ResizeAware>
     );
-  }
+  };
 
   /**
    * Renders the react component for pagination.
    *
    * @returns {XML|null}
    */
-  renderPagination() {
-    const totalCount = this.state.totalCount;
-    const limit = this.props.limit;
-    if (this.state.versions.length === 0 || totalCount <= limit) {
+  const renderPagination = () => {
+    if (versions.length === 0 || totalCount <= limit) {
       return null;
     }
-    const props = {
+    const paginatorProps = {
```

Comment:
Renamed but there's no global `prop` var for it to clash with - not clear why this was changed? Especially since it wasn't renamed in `renderVersionDetail`?

Leave as is - but I think for future PRs in this chain of refactors if there's changes like this which you haven't pointed out I'll stop reviewing and send it back with a "please point out all the changes" comment.